### PR TITLE
Ask for feedback

### PR DESCRIPTION
**Pull request by pellared**
Part of https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/471

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,12 +4,22 @@
...
 Meeting notes are available as a public [Google
 doc](https://docs.google.com/document/d/1XedN2D8_PH4YLej-maT8sp4RKogfuhFpccRi3QpUcoI/edit?usp=sharing).
 
+Get in touch on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
+If you are new, you can create a CNCF Slack account [here](http://slack.cncf.io/).
+
+## Submit an experience report
+
+We are in early phase of development
+and we are looking forward for getting any experience report.
```
Comment (pjanotti) [inline docs/CONTRIBUTING.md:17]:
```suggestion
and we are looking for your feedback.
```

File: docs/README.md [hunk 2]
```diff
@@ -37,6 +37,15 @@ This project is in the early stages of development.
 [The project board](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/projects/1)
 shows the current work in progress.
 
+⚠️ **We need you!** ⚠️
+
+Please leave us an **experience report** (in whatever form you like).
```
Comment (pjanotti) [inline docs/README.md:42]: Leave you feedback
```suggestion
Please, give us your **feedback** (in whatever form you like).
```

**Review by pjanotti — COMMENTED** [review]
"Experience report" sounds too formal to me. I prefer something like "feedback", but, won't block this because of that. See my suggestions.

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -4,12 +4,22 @@
...
-or get in touch on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
-
+Please join our weekly [SIG meeting](https://github.com/open-telemetry/community#special-interest-groups).
 Meeting notes are available as a public [Google
 doc](https://docs.google.com/document/d/1XedN2D8_PH4YLej-maT8sp4RKogfuhFpccRi3QpUcoI/edit?usp=sharing).
 
+Get in touch on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
+If you are new, you can create a CNCF Slack account [here](http://slack.cncf.io/).
+
+## Submit an experience report
```
Comment (pjanotti) [inline docs/CONTRIBUTING.md:14]: Perhaps
```suggestion
## Give feedback
```

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -4,12 +4,22 @@
...
+If you are new, you can create a CNCF Slack account [here](http://slack.cncf.io/).
+
+## Give feedback
+
+We are in early phase of development
+and we are looking for your feedback.
+
+You can do this by [submitting a GitHub issue](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/new).
+
+You may also prefer writting on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
```
Comment (nrcventura) [inline docs/CONTRIBUTING.md:21]:
```suggestion
You may also prefer writing on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
```

File: docs/README.md [hunk 5]
```diff
@@ -37,6 +37,15 @@ This project is in the early stages of development.
 [The project board](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/projects/1)
 shows the current work in progress.
 
+⚠️ **We need you!** ⚠️
+
+Please, give us your **feedback** (in whatever form you like).
+
+You can do this by [submitting a GitHub issue](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/new).
+
+You may also prefer writting on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
```
Comment (nrcventura) [inline docs/README.md:46]:
```suggestion
You may also prefer writing on [Slack](https://cloud-native.slack.com/archives/C01NR1YLSE7).
```

File: docs/README.md [hunk 6]
```diff
@@ -37,6 +37,15 @@ This project is in the early stages of development.
 [The project board](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/projects/1)
 shows the current work in progress.
 
+⚠️ **We need you!** ⚠️
```
Comment (nrcventura) [inline docs/README.md:40]: I like how this grabs my attention when I first see it.

**Review by nrcventura — COMMENTED** [review]
This looks good, other than the misspelling of `writing`.

**Review by nrcventura — APPROVED** [review]

**Pull request merged**